Sources/ComposableArchitecture/Internal/Deprecations.swift

```diff
@@ -5,6 +5,36 @@ import XCTestDynamicOverlay
 
 // NB: Deprecated after 0.38.0:
```

Comment:
I think you want to move this deprecation above and add a "Deprecated after 0.38.2" section.